docs/attributes-registry/rule.md

```diff
@@ -0,0 +1,26 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Rule
+
+## Rule Attributes
+
+Describes rule attributes. Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other notable events.
+
+| Attribute          | Type   | Description                                                                                                                                     | Examples                                                  | Stability                                                        |
+| ------------------ | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- | ---------------------------------------------------------------- |
+| `rule.author`      | string | Name, organization, or pseudonym of the author or authors who created the rule used to generate this event.                                     | `username1`                                               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
It could be difficult to add more structure to author (e.g. separate name, org, email, alias, etc).

The suggestion is to rename it  to `rule.author.id` (?) and then reserve the `rule.author.*` namespace to any additional things that may come later. 
Also it's be more clear that it's an opaque string identifying author with no special semantics.

Reply:
In the description you have provided it looks to me that it's a `user` namespace. So it would be `author` of type `user.*` instead of `string`

But I'm wondering if we should do it now. I don't know a use-case for it and it violates ECS notation if we will start to do changes here without existing use cases

Reply:
Can we postpone adding the attribute until we have a use-case?

Reply:
There are a lot of discussions on this and similar PRs where we don't know if we should reuse an existing attribute, add a new one, etc. 

I feel that if we were adding attributes along with semconvs that use them (spans, logs, metrics), we'd had much easier time understanding the scenarios and which attribute would make sense.

Regarding this attribute, based on the description `rule.author` can be absolutely anything (e.g. an organization name or an email) it feels like something that might need to be specified more precisely in the future and I want to make sure there is a room for it.